mass_mailing_list_dynamic/wizards/mailing_load_filter.py

```diff
@@ -1,12 +1,13 @@
 # Copyright 2017 Tecnativa - Pedro M. Baeza
+# Copyright 2020 Hibou Corp. - Jared Kipe
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from odoo import fields, models
 
 
-class MassMailingLoadFilter(models.TransientModel):
-    _name = "mail.mass_mailing.load.filter"
-    _description = "Mail Mass Mailing Load Filter"
+class MailingLoadFilter(models.TransientModel):
+    _name = "mailing.load.filter"
```

Comment:
@pedrobaeza Please correct me if I'm wrong: this is OK and it doesn't need a migration script because it is a transient model and there's no problem if the previous table gets dumped and a new one is created. Right?

Reply:
Yes, transient models are not mandatory to be renamed.